ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource

```diff
@@ -0,0 +1,54 @@
+*** Settings ***
+Documentation    This is a resource file for Distributed Workloads.
+Library          OperatingSystem
+Library          Process
+
+
+*** Variables ***
+${DW_TEST_TIMEOUT_LONG}    20m
+${GO_TEST_TIMEOUT}    1h
+${JOB_GO_BIN}    %{WORKSPACE=.}/go-bin
+${GO_JUNIT_REPORT_TOOL}    github.com/jstemmer/go-junit-report/v2@latest
+
+
+*** Keywords ***
+Clone Git Repository
+    [Documentation]   Clone Git repository
+    [Arguments]    ${DW_REPO_URL}    ${DW_REPO_BRANCH}    ${DW_DIR}
+    ${result} =    Run Process    git clone -b ${DW_REPO_BRANCH} ${DW_REPO_URL} ${DW_DIR}
+    ...    shell=true    stderr=STDOUT
+    Log To Console    ${result.stdout}
+    IF    ${result.rc} != 0
+        FAIL    Unable to clone DW repo ${DW_REPO_URL}:${DW_REPO_BRANCH}:${DW_DIR}
+    END
+
+Run Distributed Workloads Tests
+    [Documentation]   Run distributed workloads E2E tests from integration directory
+    [Arguments]    ${DW_DIR}    ${DW_TEST_RESULT_FILE}
+    ${result} =    Run Process
+    ...    go test -v -timeout ${GO_TEST_TIMEOUT} ./integration 2>&1 > ${DW_TEST_RESULT_FILE}
+    ...    shell=true    cwd=${DW_DIR}/tests    env:ODH_NAMESPACE=redhat-ods-applications
+    ...    env:DW_TEST_TIMEOUT_LONG=${DW_TEST_TIMEOUT_LONG}
```

Comment:
Where is the `DW_TEST_TIMEOUT_LONG` declared and what does it represent? In DW tests we use env variable `CODEFLARE_TEST_TIMEOUT_LONG` - https://github.com/project-codeflare/codeflare-common/blob/main/support/support.go#L54

Reply:
@sutaakar Thank you for noticing it! It should be fixed now.